README.md

```diff
@@ -145,16 +139,14 @@ analytics, and operations end-to-end:
   - Custom dashboards with deep analytics, alerts, and reporting
   - Expert team to consult and improve business metrics
 
-You can [try the hosted version in our sandbox][dashboard].
+You can [try the hosted version in our sandbox](). <!-- needs a link here! -->
```

Comment:
We used reference-style links to avoid duplication of links and make the source content more readable. Regardless, I couldn't find any information on the linked article that discourage the use of reference-style links. Any specific reason for changing all of them to inline links?

Reply:
The reason for inline links as when people who use accessibility settings, the audio is read out as it is. Thus the person hearing it can't just stop and "scroll" to the bottom of the page to see where/what it is linking it. Having the link directly attached to where it's referenced helps these people quickly navigate. It's up to you thought :wink: let me know what you'd like to do.